playbooks/openstack-caasp_instance.yml

```diff
@@ -193,20 +215,10 @@
 
         - meta: refresh_inventory
 
-        - name: Wait for a SSH to be properly started
-          wait_for:
-            host: "{{ hostvars[item]['ansible_host'] }}"
-            port: 22
-            search_regex: OpenSSH
-            state: started
-            delay: 10
-            timeout: 180
-          loop: "{{ groups['caasp-admin'] + groups['caasp-masters'] + groups['caasp-workers'] }}"
-
         - name: Get pubkey, and add it to known hosts
           command: "ssh-keyscan -H {{ hostvars[item]['ansible_host'] | quote }}"
           register: _caaspsshkeys
-          retries: 10
+          retries: 25
```

Comment:
Why have we gotten rid of the wait for ssh?

Reply:
because it has no purpose?

The only reason was to wait_for ssh to be up before scanning keys. But since then we have a `until:` condition that ensures the same things.